python/pyhdk/__init__.py

```diff
@@ -20,6 +20,7 @@
 import pyhdk.storage as storage
 from pyhdk.hdk import init
 from pyhdk._builder import QueryBuilder
+from pyhdk.version import Version 
```

Comment:
It is very common for Python packages to have `__version__` string attribute. So I think we should provide `pyhdk.__version__` attribute holding `pyhdk.Version.str()` value. But note that it differs from the version we use in the `setup.py` (`@HDK_VERSION_RAW@`). I don't know if it can cause any problems though.